### PR TITLE
Change 'requested' to 'launched' in evaluations ui

### PR DESCRIPTION
**Pull request by johannkm**
"Requested" is a confusing word in the evaluations UI. When we display `1 requested / 1 skipped / 0 discarded` it seems like a partition was requested, but then skipped. It makes sense to think that way because the materialize conditions can be overridden by skips- e.g. I could say that in an evaluation where a materialization condition was met, we "requested" a run but then had that run discarded or skipped.

The two alternatives we've discussed are "materialized" and "launched". The problem with "materialized" is that while we kicked off a materialization, it may fail. The issue with launched is we don't usually use it with assets- on the other hand we still display `Launched run xyz` once you click materialize. I think it's an intuitive enough term and clearer than requested.

Screenshot 2023-07-11 at 8 32 29 AM
Screenshot 2023-07-11 at 8 32 06 AM

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #15218** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/15218" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/15218?utm_source=stack-comment).

**Review by braunjj — COMMENTED** [review]
LGTM

**Review by hellendag — APPROVED** [review]

**Comment by johannkm** [comment]
> Is a run considered "launched" if it's in the queue? If not, should we say "submitted"?

This has been an issue ever since I added the run coordinator. Overall I think 'submitted' was a mistake in that a `launch` button is more clear than a `submit` button, so we've kept it as `launch` when under the hood it actually does a submit. We haven't come up with another way to differentiate the two. Maybe one day submitted will turn in to launched and current day launched (making some api/cli call to start a run worker) will be called 'starting' (to match the run status) or something.

Long story short, we already use `launched` in this context in a few other places and I think we'll continue to do so

**Comment by sryza** [comment]
@johannkm ok sounds good. Easy to change in the future also if for some reason we need to.
